# HG changeset patch
# User jwe
# Date 1140454298 0
#      Mon Feb 20 16:51:38 2006 +0000
# Node ID 53d15d4c8dd7e48cc188b68ed1a4b47d763a5c65
# Parent  0f6a0a85a857e0107e0544cf0459bde4bb305712
[project @ 2006-02-20 16:51:38 by jwe]
README.mirrors

diff --git a/README.mirrors b/README.mirrors
--- a/README.mirrors
+++ b/README.mirrors
@@ -3,29 +3,27 @@ one of those listed below.  They mirror 
 ftp.octave.org
 
 FTP:
 
   site                          directory
   ----                          ---------
   ftp.eos.hokudai.ac.jp         /pub/GNU/misc/octave
   ftp.math.uni-hamburg.de       /pub/soft/math/octave
-  ftp.task.gda.pl               /pub/software/octave
   ftp.u-aizu.ac.jp              /pub/SciEng/numanal/Octave
-  mirror.aarnet.edu.au          /pub/octave
   neacm.fe.up.pt                /pub/octave
 
 HTTP:
 
   url
   ---
   http://neacm.fe.up.pt/pub/octave
+  http://octave.oss-mirror.org
+  http://octave.linux-mirror.org
 
-The latest Octave sources may also be found on the official Free
-Software Foundation ftp site, ftp://ftp.gnu.org/pub/gnu/octave, which is
-also mirrored on many sites around the world.
 
 John W. Eaton
 jwe@bevo.che.wisc.edu
 University of Wisconsin-Madison
 Department of Chemical & Biological Engineering
 
-Thu Jun  3 14:37:06 2004
+Mon Feb 20 11:51:09 2006
+
